get-sysinternals-du.test.js

```diff
@@ -0,0 +1,64 @@
+if (process.platform !== 'win32') {
+  // these test cases are only for win32
+  return
+}
+
+const { test, beforeEach } = require('tap')
+const path = require('path')
+const os = require('os')
+const fs = require('fs')
+const subject = require('./get-sysinternals-du.js')
+
+let workspace = path.join(os.tmpdir(), 'fast-folder-size-playground')
+beforeEach(() => {
+  if (fs.existsSync(workspace)) fs.rmSync(workspace, { recursive: true })
+  fs.mkdirSync(workspace)
+  subject.workspace = workspace
+})
+
+test('it can use local file path as process.env.FAST_FOLDER_SIZE_DU_ZIP_LOCATION', t => {
+  let dummyDuZipPath = path.join(workspace, 'dummy-du.zip')
+  fs.writeFileSync(dummyDuZipPath, '')
+  process.env.FAST_FOLDER_SIZE_DU_ZIP_LOCATION = dummyDuZipPath
+
+  subject.onDuZipDownloaded = function (tempFilePath) {
+    t.equal(dummyDuZipPath, tempFilePath)
+    t.end()
+  }
+
+  subject.default()
+})
+test('it cannot use non-exists local file path as process.env.FAST_FOLDER_SIZE_DU_ZIP_LOCATION', t => {
+  process.env.FAST_FOLDER_SIZE_DU_ZIP_LOCATION = path.join(
+    workspace,
+    'non-exists-dummy-du.zip'
+  )
+
+  t.throws(subject.default, error => {
+    return error.message.startsWith('du.zip not found at')
+  })
+  t.end()
```

Comment:
when you don't need to assert asynchronously in callbacks, please write the test function as an async function so you don't have to remember calling t.end

Reply:
![image](https://github.com/simoneb/fast-folder-size/assets/16363180/a1871d46-dce7-4e3a-a925-1e0214054420)

Once I added the `async` keyword, eslint shows an error on the parameter `t`: `ESLint: Parsing error: Unexpected token t`.
Everything looks fine to me as I'm following the doc https://node-tap.org/docs/api/promises/ , do you know how to fix this ?

Reply:
probably just an eslint configuration to support esm, should be easy to address